WordPress/Classes/ViewRelated/NUX/UnifiedProloguePages.swift

```diff
@@ -0,0 +1,78 @@
+import UIKit
+
+enum UnifiedProloguePageType: CaseIterable {
+    case intro
+    case editor
+    case notifications
+    case analytics
+    case reader
+
+    var title: String {
+        switch self {
+        case .intro:
+            return NSLocalizedString("Welcome to the world's most popular website builder.", comment: "Caption displayed in promotional screens shown during the login flow.")
+        case .editor:
+            return NSLocalizedString("With this powerful editor you can post on the go.", comment: "Caption displayed in promotional screens shown during the login flow.")
+        case .notifications:
+            return NSLocalizedString("See comments and notifications in real time.", comment: "Caption displayed in promotional screens shown during the login flow.")
+        case .analytics:
+            return NSLocalizedString("Watch your audience grow with in-depth analytics.", comment: "Caption displayed in promotional screens shown during the login flow.")
+        case .reader:
+            return NSLocalizedString("Follow your favourite sites and discover new reads.", comment: "Caption displayed in promotional screens shown during the login flow.")
+        }
+    }
+}
+
+/// Simple container for each page of the login prologue.
+///
+class UnifiedProloguePageViewController: UIViewController {
+    private let stackView = UIStackView()
+    private let titleLabel = UILabel()
+
+    private var pageType: UnifiedProloguePageType!
+
+    init(pageType: UnifiedProloguePageType) {
+        self.pageType = pageType
+
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        view.backgroundColor = .clear
+
+        configureStackView()
+        configureTitle()
```

Comment:
since we are not using IB, I wonder if we should build/configure the view in `loadView` instead of `viewDidLoad`? What do you think?

Reply:
Sure, happy to do either – I just kind of default to `viewDidLoad` 🙂